dlt/load/load.py

```diff
@@ -112,18 +122,23 @@ def spool_new_jobs(self, load_id: str, schema: Schema) -> Tuple[int, List[LoadJo
         # remove None jobs and check the rest
         return file_count, [job for job in jobs if job is not None]
 
-    def retrieve_jobs(self, client: JobClientBase, load_id: str) -> Tuple[int, List[LoadJob]]:
+    def is_staging_job(self, file_path: str) -> bool:
+        return self.staging is not None and file_path.split(".")[-1] in self.staging.capabilities().supported_loader_file_formats
```

Comment:
use `splitext`

Reply:
or it would be way better if you use `ParsedLoadJobFileName.parse` whenever you need a file format out of a job file. could we do that?